Python/Embeddings/RegExp (for Python).sublime-syntax

```diff
@@ -25,21 +30,52 @@ contexts:
 
   group-start:
     - meta_prepend: true
-    - match: (\?P=)([a-zA-Z_][a-zA-Z_0-9]*\w*)(?=\))
+    - match: (\?P=)({{capture_group_name}})(?=\))
       captures:
         1: keyword.other.back-reference.named.regexp
         2: variable.other.backref-and-recursion.regexp
-    - match: (\?P)(<)([a-z]\w*)(>)
```

Comment:
This part of the change is significant. Capture groups can start with capital letters or underscores, too.